Dev/config/FirebaseFunctions.js

```diff
@@ -423,9 +426,31 @@ export default class FirebaseFunctions {
             ID: studentID
         });
         const student = await this.getStudentByID(studentID);
-        const finalObject = await this.joinClass(student, classID);
 
-        return finalObject;
+        const classToJoin = await this.classes.doc(classID).get();
+        if (!classToJoin.exists) {
+            return -1;
```

Comment:
What does the UI show in this case? Should this be an exception instead to ensure that the caller always handles it?